var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -135,6 +137,33 @@ def patch(self):
             # Disable obsolete workaround
             filter_file('ifdef __PGI', 'if 0', 'src/gromacs/fileio/xdrf.h')
 
+        if '+cuda' in self.spec:
+            # Upstream supports building of last two major versions of Gromacs.
+            # Older versions of Gromacs need to be patched to build with more recent
+            # versions of CUDA library.
+
+            # Hardware version 3.0 is supported up to CUDA 10.2
+            # (Gromacs 4.6-2019 needs to be patched, 2020.4 is handling it correctly)
+
+            if self.spec.satisfies('@4.6:2020.3^cuda@11:'):
```

Comment:
Please update the comment to address Marvin's issue.

Reply:
Done. Sorry for the delay, it was night at my place :)

Reply:
Np.  Thanks for the fix.